libdnf5-cli/argument_parser.cpp

```diff
@@ -1048,6 +1048,7 @@ void ArgumentParser::CommandOrdinary::parse(const char * option, int argc, const
         }
     }
     size_t used_positional_arguments = 0;
+    char unused_short_option = 0;
```

Comment:
The `unused_short_option` variable is used only within the following `for` loop. I’d prefer to define it inside the loop so that its value isn’t carried across iterations and is clearly reset for each processed argument.